db/migrate/20200602170103_rename_groups_to_folders.rb

```diff
@@ -0,0 +1,8 @@
+class RenameGroupsToFolders < ActiveRecord::Migration[6.0]
```

Comment:
I added this migration. did you forget to check this in ?

Reply:
Yes - sorry for that.